manim/mobject/mobject.py

```diff
@@ -641,11 +885,45 @@ def resume_updating(self, recursive=True):
 
     # Transforming operations
 
-    def apply_to_family(self, func):
+    def apply_to_family(self, func) -> "Mobject":
```

Comment:
`func` is a function that accepts Mobject as a parameter and doesn't return anything right?
```suggestion
    def apply_to_family(self, func: Callable[["Mobject"], None]) -> "Mobject":
```